### PR TITLE
dev-db/victoria-metrics: Initial import

### PR DESCRIPTION
**Pull request by patricklauer**
ebuild by me

Package-Manager: Portage-2.3.99, Repoman-2.3.22

### CODE REVIEW

File: dev-db/victoriametrics/victoriametrics-1.35.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+BDEPEND="dev-lang/go"
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${P}/src/${PN}"
+
+src_compile() {
+	# need to explicit because makefile does a docker by default
+	use cluster &&	emake -j8 vmagent vmalert vmbackup vmrestore vminsert vmselect vmstorage
+	use cluster ||  emake -j8 victoria-metrics vmagent vmalert vmbackup vmrestore
```
Comment (alip) [inline dev-db/victoriametrics/victoriametrics-1.35.0.ebuild:31]: Nitpick: jobs hardcoded.

File: dev-db/victoriametrics/victoriametrics-1.35.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+src_install() {
+	# what do you mean make install? you crazy?!
+	mkdir -p "${D}/usr/bin"
+	cp bin/* "${D}/usr/bin"
```
Comment (alip) [inline dev-db/victoriametrics/victoriametrics-1.35.0.ebuild:37]: Missing die clauses.
Good idea to ensure permissions of the installed binaries.

**Review by alip — APPROVED** [review]
Looks good overall, please fix the minor QA issues upon merge.

**Comment by xmhd** [comment]
Is this still required?

**Comment by xmhd** [comment]
closing for now, can re-open if required

**Comment by mcharriere** [comment]
😭 I'm about to send an update for this, stay tuned

**Comment by xmhd** [comment]
> sob I'm about to send an update for this, stay tuned

no worries, we can get this into the overlay if required.

**Review by xmhd — APPROVED** [review]

**Pull request merged**